pkg/peertls/utils.go

```diff
@@ -22,6 +22,32 @@ import (
 	"storj.io/storj/pkg/pkcrypto"
 )
 
+// NonTemporaryError is an error with a `Temporary` method which always returns false.
+// It is intended for use with grpc.
+//
+// (see https://godoc.org/google.golang.org/grpc#WithDialer
+// and https://godoc.org/google.golang.org/grpc#FailOnNonTempDialError).
+type NonTemporaryError struct {
```

Comment:
Same here, should this be in `pkg/transport`?

Reply:
:+1: